docs/using-the-jdbc-driver/DataSource.md

```diff
@@ -77,6 +77,8 @@ To use the AWS JDBC Driver with a connection pool, you must:
    targetDataSourceProps.setProperty("serverName", "db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com");
    targetDataSourceProps.setProperty("databaseName", "postgres");
    targetDataSourceProps.setProperty("port", "5432");
+   
+   ds.addDataSourceProperty("targetDataSourceProps", targetDataSourceProps);
```

Comment:
Property name should be "targetDataSourceProperties"

Reply:
Addressed in https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/213/commits/f23fd3aa62a806dc2b603934a3df26278e294617